### DISCUSSION TITLE
"New Manual"

### DISCUSSION THREAD

**Discussion by dalers** [timschofield/webERP#245] [Problems / Bugs]
2024-11-20 re-titled to have general discussion on "the new manual."

Original post: I was checking to see if the manual image issue #231 was resolved (which it was and I closed the isssue), and I noticed the button for the user manual does not always appear at the top of every page (which imho should be the case). What is the rationale for removing the buttom from the main page and possibly other pages?

**Comment by dalers** [comment]
Point 2. I REALLY like the contextual user's manual, but so far I'm not a fan of the "new manual" in-app experience. I don't like I can't have the manual open in one window as reference while I'm entering data into a form, and I can't actually enter the data until I close the manual. I admit I actually need to read your documentation on the users manual design though...

Also, what is the relationship between the documentation on the website (https://weberp.org/Documentation.html) and the users manual included with webERP? Does the online documentation on weberp.org somehow derive from the users manual provided with webERP? If so, how? Would it be possible to also generate the former "in-browser" format manual? 

Point 3. Who is the user of the manual? Is it time for a seperate users manual and developers manual? Perhaps a manual in layers, with a user top user layer linking to lower level technical information where appropriate. My first impression as a typical business system user or power user, was that there is a lot of in-line code and sysadmin/developer information obscuring more basic "how to config x" or "how to do y" information a user would want.

**Comment by timschofield** [comment]
Simple answer is that I removed it :)

Long answer is that I was unsure what to put there for the menu pages
as it is supposed to be context sensitive. It did have the manual
index page, but that didn't work as the links didn't function as you
have pointed out elsewhere. So as a short term answer I removed it.

Thinking about this as I type, how about when on the index page, not
using the context system at all, and actually creating a new browser
tab for the index page as used to happen, and just using the new
system when using specific functionality?

Tim

On Mon, 18 Nov 2024 at 15:46, Dale Scott ***@***.***> wrote:
>
> I was checking to see if the manual image issue #231 was resolved (which it was and I closed the isssue), and I noticed the button for the user manual does not always appear at the top of every page (which imho should be the case). What is the rationale for removing the buttom from the main page and possibly other pages?
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>



-- 
www.weberp.org
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by timschofield** [comment]
Hi Dale, I have changed includes/header.php so that now when you are on the
menu page it shows the manual link. Clicking at it from there takes you to
the index page in a new tab. The new "context sensitive" help should carry
on working as before.

Thanks
Tim


On Wed, 20 Nov 2024 at 16:58, Dale Scott ***@***.***> wrote:

> Point 2. I REALLY like the contextual user's manual, but I'm not a fan so
> far of the in-app-type manual (what is it technically? in-document?). I
> don't like I can't have the manual open in one window as reference for the
> form I'm entering data in, and not be able to enter data without closing
> the help. I need to actually your documentation on the users manual design,
> but is it possible to still use the manual directly in a seperate browser?
>
> Seperately, what is the relationship between the documentation on the
> website (https://weberp.org/Documentation.html) and the users manual
> included with webERP? Does the online documentation on weberp.org somehow
> derive from the users manual provided with webERP? If so, how?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/245#discussioncomment-11326235>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LFOI5NMUKL2CC5ZOBD2BS5VDAVCNFSM6AAAAABR7647LWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMZSGYZDGNI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by dalers** [comment]
Hi Tim, I think that's a reasonable solution so a user can always open a stand-alone manual from the Main Menu (and clicking the Users Manual button in any page _except_ the Main Menu opens context sensitive "in-app" help). How about I add a line to the top of the stand-alone manual to explain this? What words do _you_ use to describe the two different types of manuals? (or is there just one manual with two ways to access it?)

How does the users manual provided with webERP relate to the manual on the weberp.org site? Do they come from the same source?

Also, the "Display Checked" functionality in the stand-alone manual isn't working, at least for me. E.g. selecting Introduction and clicking "Display Checked" doesn't stay "in the manual" but shows a webERP page with "ERROR Report : This form was not submitted with a correct ID"

![image](https://github.com/user-attachments/assets/12a34fbb-af08-4891-aba5-affcb3171f46)

**Comment by timschofield** [comment]
To be honest the display checked isn't necessary at all. Each section
has it's own link and I think that's enough.

The manual is all from the same source (website, in page help, and
manual from the menu page), just different ways of accessing the same
document.

Adding that line to ManualContents.php sounds like a good idea.

Thanks
Tim

On Sat, 23 Nov 2024 at 23:34, Dale Scott ***@***.***> wrote:
>
> Hi Tim, I think that's a reasonable solution so a user can always open a stand-alone manual from the Main Menu (and clicking the Users Manual button in any page except the Main Menu opens context sensitive "in-app" help). How about I add a line to the top of the stand-alone manual to explain this? What words do you use to describe the two different types of manuals? (or is there just one manual with two ways to access it?)
>
> How does the users manual provided with webERP relate to the manual on the weberp.org site? Do they come from the same source?
>
> Also, the "Display Checked" functionality in the stand-alone manual isn't working, at least for me. E.g. selecting Introduction and clicking "Display Checked" doesn't stay "in the manual" but shows a webERP page with "ERROR Report : This form was not submitted with a correct ID"
>
> image.png (view on web)
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.Message ID: ***@***.***>



-- 
www.weberp.org
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by dalers** [comment]
I suspect the "Display Checked" feature was for printing sections of the manual of interest. 

Does anyone have any interest in this feature or in having it fixed?

If not, I suggest the "Display Checked" feature of the users manual be removed as it is an obvious fail for a new user and contirubutes negatively to credibility (and even if worked, imho would not be a significant convenience compared to printing the subsections individually)
